artifacts/definitions.py

```diff
@@ -22,24 +25,32 @@
     'Containerd': 'Containerd artifacts',
     'Docker': 'Docker artifacts.',
     'Execution': 'Contain execution events.',
-    'ExternalAccount': (
+    'ExternalAccount': '<Deprecated>', # TODO: to be deleted
+    'External Account': (
         'Information about any user accounts e.g. username, '
         'account ID, etc.'),
     'External Media': 'Contain external media data or events e.g. USB drives.',
+    'File System': 'Artifacts related to the file system e.g. MFT.',
     'Hadoop': 'Hadoop artifacts.',
     'IM': 'Instant Messaging / Chat applications artifacts.',
+    'Interactive': 'Artifacts created by interactive (e.g. RDP) user activity.',
     'iOS': 'Artifacts related to iOS devices connected to the system.',
     'History Files': 'History files artifacts e.g. .bash_history.',
-    'KnowledgeBase': 'Artifacts used in knowledge base generation.',
+    'KnowledgeBase': '<Deprecated>', # TODO: to be deleted
+    'Knowledge Base': 'Artifacts used in knowledge base generation.',
     'Kubernetes': 'Kubernetes artifacts',
     'Logs': 'Contain log files.',
     'Mail': 'Mail client applications artifacts.',
     'Memory': 'Artifacts retrieved from memory.',
     'Network': 'Describe networking state.',
+    'Persistence' : 'Persistence mechanisms e.g. the Startup folder.',
     'Plist': 'Artifact that is a plist.',
     'Processes': 'Describe running processes.',
     'Rekall': 'Artifacts using the Rekall memory forensics framework.',
-    'Software': 'Installed software.',
+    'Registry': 'Files related to the Windows Registry.',
+    'Security Agents': 'Endpoint detection and response related artifacts, '
+        'e.g. antivirus quarantine files.',
+    'Software': 'Information about installed software.',
```

Comment:
How is this label different from Applications ?

Reply:
The distinction I was trying to make is that "Applications" relates to artifacts related directly to software, such as logs and configuration files, while "Software" would relate to what software was installed / software inventory (e.g. Python packages, Chrome extensions, the Uninstall key in the Windows registry, etc.). Maybe there's a better way to draw that distinction by renaming "Software" to "Software Inventory", WDYT?

Reply:
But what would be the use case, I think labels are mainly used in GRR at the moment and by the stats script https://artifacts.readthedocs.io/en/latest/sources/background/Stats.html. Maybe there are other ways / categories that could be more fitting